packages/plugin-core/src/_extension.ts

```diff
@@ -336,6 +336,7 @@ export async function _activate(
         wsRoot: maybeWsRoot,
         engine: resp.data.engine,
         wsService,
+        opts,
```

Comment:
@kevinslin we were not passing in the skip options to workspace activator so all tests were initializing the tree view. This caused the tests that would otherwise run fast to time out.